keras/src/backend/tensorflow/core.py

```diff
@@ -206,6 +206,20 @@ def cond(pred, true_fn, false_fn):
     return tf.cond(pred, true_fn=true_fn, false_fn=false_fn)
 
 
+def correlate(a, v, mode):
+    if isinstance(a, np.ndarray):
+        a = a.tolist()
```

Comment:
This implementation uses Python rather than TensorFlow, so it will only be runnable eagerly, and it cannot be differentiated. It is also inefficient. You should instead use `conv` to implement correlation.

See example: https://jax.readthedocs.io/en/latest/_modules/jax/_src/numpy/lax_numpy.html#correlate